sandbox.go

```diff
@@ -424,26 +424,56 @@ func (sb *sandbox) ResolveName(name string) net.IP {
 	parts := strings.Split(name, ".")
 	log.Debugf("To resolve %v", parts)
 
+	reqName := parts[0]
+	networkName := ""
+	if len(parts) > 1 {
+		networkName = parts[1]
+	}
+
+	// First check for local container alias
+	ip = sb.resolveName(reqName, networkName, true)
```

Comment:
May be pass the result of sb.getConnectedEndpoints() as a param rather traversing it twice